SlicerDWD/SlicerDWD.py

```diff
@@ -20,12 +20,20 @@
 
 import sklearn.metrics
 
+import importlib
+
+DWD_VERSION = '1.0.2'
+
 try:
     import dwd
 except ModuleNotFoundError:
-    slicer.util.pip_install("dwd")
+    slicer.util.pip_install('dwd=={}'.format(DWD_VERSION))
     import dwd
 
+if dwd.__version__ != DWD_VERSION:
+    slicer.util.pip_install('dwd=={}'.format(DWD_VERSION))
+    importlib.reload(dwd)
```

Comment:
It may be sensible to report a warning suggesting the user to restart as reloading may not lead to have all the package dependencies also reloaded